tests/pipelines/unclip/test_unclip_image_variation.py

```diff
@@ -491,6 +491,9 @@ def test_save_load_local(self):
     def test_save_load_optional_components(self):
         return super().test_save_load_optional_components()
 
+    def test_float16_inference(self):
+        super().test_float16_inference(expected_max_diff=1.0)
```

Comment:
```suggestion
        ...
```

think values can not have a higher diff than 1.0 so this test is a bit moot I think. Let's maybe just skip it.